src/test_command_line_parser.cc

```diff
@@ -1657,40 +1657,6 @@ TEST_CASE("Testing Command Line Parser")
       exp->measurement_mode = MeasurementMode::COUNT_WINDOWS;
       exp->measurement_request_count = 2000;
     }
-    SUBCASE("zero value (no override to measurement mode)")
-    {
-      int argc = 7;
-      char* argv[argc] = {app_name,          "-m", model_name,
-                          "--request-count", "0",  "--measurement-mode",
-                          "time_windows"};
-
-      REQUIRE_NOTHROW(act = parser.Parse(argc, argv));
-      CHECK(!parser.UsageCalled());
-
-      exp->request_count = 0;
-      exp->measurement_mode = MeasurementMode::TIME_WINDOWS;
-    }
-    SUBCASE("zero value (no override to measurement request count)")
-    {
-      int argc = 9;
-      char* argv[argc] = {
-          app_name,
-          "-m",
-          model_name,
-          "--request-count",
-          "0",
-          "--measurement-mode",
-          "count_windows",
-          "--measurement-request-count",
-          "50"};
-
-      REQUIRE_NOTHROW(act = parser.Parse(argc, argv));
-      CHECK(!parser.UsageCalled());
-
-      exp->request_count = 0;
-      exp->measurement_mode = MeasurementMode::COUNT_WINDOWS;
-      exp->measurement_request_count = 50;
-    }
```

Comment:
This was testing behavior around user passing `--request-rate=0` but really that shouldn't have been allowed.

It errors now, as the command_line_parser [message](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#r2057071264) said all along.